demos/demo_trifinger_impedance_control.py

```diff
@@ -0,0 +1,106 @@
+import math
+import time
+import pinocchio
+import numpy as np
+
+from robot_wrapper import Robot
+
+import blmc_robots
+import robot_interfaces
+
+
+# Initialize pinocchio model
+robot = Robot(visualizer='gepetto',
+                  viscous_friction=0.0)
+robot.initViewer(loadModel=True)
+
+robot_model = robot.model
+robot_data = robot_model.createData()
+
+# Initialize real robot
+finger = blmc_robots.Robot(robot_interfaces.trifinger,
+                           blmc_robots.create_trifinger_backend,
+                           "trifinger.yml")
+finger.initialize()
+
+# End effector Ids
+finger_tip_links = ["finger_tip_link_0", "finger_tip_link_120", "finger_tip_link_240"]
+finger_tip_ids = [robot_model.getFrameId(name) for name in finger_tip_links]
+
+
+def next_target(target_type, i=None, t=None, idx=None):
+	# defining step targets
+	if target_type == "step":
+		z = [0.05, 0.2]
+		if t % 1001 == 0:
+			idx = (idx + 1) % 2
+		return z[idx], idx
+
+	# defining continuous, more smoother trajectory
+	if target_type == "continuous":
```

Comment:
Should be an `elif` here.  Also the `else` case (i.e. invalid value of `target_type`) should be handled; probably simply by raising a ValueError.